docs/rstudio.md

```diff
@@ -74,64 +70,75 @@ X['colour'] <- as.numeric(X['colour'] == "Black")
 y <- as.numeric(df$checks > 1)
 ```
 
-In this case we're taking a subset of the `load_arrests` dataset. This information
-contains arrests data and we're going to perform gridsearch keeping fairness in mind. 
+In this case we're taking a subset of the `load_arrests` dataset. This information contains arrests data and we're going
+to perform gridsearch keeping fairness in mind.
 
 ```r
 cv <- grid(
-  estimator = pipe, 
-  param_grid = list("filter__alpha"=seq(0.1, 0.9, 0.1), 
+  estimator = pipe,
+  param_grid = list("filter__alpha"=seq(0.1, 0.9, 0.1),
                     "model__threshold"=seq(0.1, 0.9, 0.1)),
   cv=as.integer(3),
-  scoring = list(accuracy=make_scorer(accuracy), 
+  scoring = list(accuracy=make_scorer(accuracy),
                  pp_score=pp_score('colour')),
   refit="accuracy")
 
 cv$fit(X, y)
 ```
 
-You'll note that we're using `list` and `as.integer` here. For details on why, 
-check out this [documentation page]().
+You'll note that we're using `list` and `as.integer` here. For details on why, check out this [documentation page]().
```

Comment:
One question I had regarding this tutorial. Here there is a missing link that I am not sure what should refer to 🙃

Reply:
This is probably fine: https://rstudio.github.io/reticulate/articles/calling_python.html#lists-tuples-and-dictionaries